### PR TITLE
Feature/add additional terms

### PR DESCRIPTION
**Pull request by samleeflang**
- Added verbatim terms
- Added ID terms
Both are needed to comply with MIDS 
https://naturalis.atlassian.net/browse/DD-993

### CODE REVIEW

**Review by southeo — DISMISSED** [review]
The word "verbatim" has ceased to feel real :dizzy: 
nice work tho

**Comment by samleeflang** [comment]
Updated to the latest version of the openDS which have now been merged into the openDS repo

File: src/main/resources/json-schema/location.json [hunk 1]
```diff
@@ -172,7 +172,7 @@
         "under water since 2005"
       ]
     },
-    "georeference": {
+    "???:GeoReference": {
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply src/main/resources/json-schema/location.json:175]: Nope at the moment GeoReference isn't a dwc class but these terms fall beneath Location
https://dwc.tdwg.org/list/#31-index-by-term-name

File: src/main/java/eu/dissco/core/translator/terms/BaseDigitalObjectDirector.java [hunk 2]
```diff
@@ -207,11 +207,11 @@ public abstract class BaseDigitalObjectDirector {
...
     var ds = assembleDigitalSpecimenTerms(data, dwc);
-    ds.withOccurrences(assembleOccurrenceTerms(data, dwc));
+    ds.withDwcOccurrence(assembleOccurrenceTerms(data, dwc));
     ds.withDwcIdentification(assembleIdentifications(data, dwc));
-    ds.withIdentifiers(assembleIdentifiers(data));
-    ds.withCitations(assembleSpecimenCitations(data, dwc));
-    ds.withEntityRelationships(assembleDigitalSpecimenEntityRelationships(ds));
+    ds.withIdentifier(assembleIdentifiers(data));
+    ds.withCitation(assembleSpecimenCitations(data, dwc));
+    ds.withEntityRelationship(assembleDigitalSpecimenEntityRelationships(ds));
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply src/main/java/eu/dissco/core/translator/terms/BaseDigitalObjectDirector.java:214]: Yes I know but then we can't use the dwc terms. What we could do is create our own `ods:Ocurrences` which then contain `dwc:Occurrence`. ABCD(EFG) uses these kind of structures, but they always felt a bit unnecessary as they add this additional level in between. Might be better though, as the `dwc:Occurrence` isn't an array. I will create a task for this

[the remark this answers is not in the document]
Comment (samleeflang) [reply src/main/java/eu/dissco/core/translator/terms/BaseDigitalObjectDirector.java:214]: https://naturalis.atlassian.net/browse/DD-1165

**Pull request merged**